cs_tools/cli/tools/git/branches.py

```diff
@@ -0,0 +1,222 @@
+import typer
+from httpx import HTTPStatusError
+from rich.align import Align
+from rich.table import Table
+
+from cs_tools.cli.dependencies import thoughtspot
+from cs_tools.cli.types import MultipleChoiceType
+from cs_tools.cli.ux import CSToolsApp, rich_console
+from cs_tools.types import DeployType, DeployPolicy
+
+app = CSToolsApp(
+    name="branches",
+    help="Tools for working with git branches and commits.",
+    add_completion=False,
+    invoke_without_command=True,
+    no_args_is_help=True,
+)
+
+# consider moving to types.
+VALID_METADATA_COMMIT_TYPES = ["LOGICAL_TABLE", "PINBOARD_ANSWER_BOOK", "QUESTION_ANSWER_BOOK"];
+
+
+@app.command(dependencies=[thoughtspot], name="commit")
+def branches_commit(
+        ctx: typer.Context,
+        org: str = typer.Option(None, help="the org to use if any"),
+        tag: str = typer.Option(None, help="the tag for metadata to commit"),
+        metadata_ids: str = typer.Option("",
+                                     custom_type=MultipleChoiceType(),
+                                     help="the metadata GUIDs or names to commit"),
+        branch_name: str =
+        typer.Option(None,
+                     help="the branch name to use for the git repository (or use the default if not provided"),
+        comment: str = typer.Option(..., help="the comment to use for the commit"),
+        delete_aware: bool = typer.Option(False, help="deletes content that doesn't exist in TS from the repo"),
```

Comment:
This argument doesn't get used, what's it supposed to do?

Reply:
It checks allows commits to remove things in GitHub.  It was in the v2 API, but wasn't getting passed.  Added.